website/integrations/services/engomo/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate with engomo
+sidebar_label: engomo
+---
+
+# Integrate with engomo
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is engomo
+
+> engomo is an low-code app development platform to create enterprise apps for smartphones and tablets based on Android, iOS, or iPadOS.
+>
+> -- https://engomo.com/
+>
+> This guide explains how to set up engomo to use authentik as the OAuth provider for the application login on the smartphone/tablet and login to the admin WebGUI (composer).
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `engomo.company` is the FQDN of the engomo installation.
+- `authentik.company` is the FQDN of the authentik installation.
+- `engomo.mapping` is the name of the Scope Mapping.
+
+:::note
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+## authentik configuration
+
+In authentik, create a new scope mapping. To do so, log in and navigate to the Admin interface, then go to **Customization --> Property Mapping** and click **Create**.
+
+- `engomo.mapping` is the value of the Mapping's name.
+- `profile` is the value for the Scope name.
+- `return {"preferred_username": request.user.email}` is the value for the Expression.
```

Comment:
What do we mean by "the Expression"? Is there an expression somewhere that defines these values?